khmer/khmer_args.py

```diff
@@ -60,9 +60,12 @@ def build_counting_args(descr=None, epilog=None):
     return parser
 
 
-def build_hashbits_args(descr=None, epilog=None):
+def build_hashbits_args(descr=None, epilog=None, parser=None):
     """Build an ArgumentParser with args for hashbits based scripts."""
-    parser = build_hash_args(descr=descr, epilog=epilog)
+    if parser is None:
+        parser = build_hash_args(descr=descr, epilog=epilog)
+    else:
+        parser = build_hash_args(descr=descr, epilog=epilog, parser=parser)
```

Comment:
since `build_hash_args` has the same default for `parser`, you can just pass it and avoid checking if it is None.